index.js

```diff
@@ -39,10 +39,10 @@ async function installDependencies(dirName) {
 
 async function initGit(dirName) {
   console.log(`Setting up Git ...`);
-  await run(`rm -rf ${dirName}/.git`);
+  // remove .git folder
+  await fs.removeSync(`${dirName} / .git`);
```

Comment:
Does this work? The syntax seems wrong. There should not be spaces in directory paths.

Reply:
This is because of my formatter, there shouldn't be any space. Thanks for enforcing the formatting.